CMakeModules/ODInternal.cmake

```diff
@@ -236,3 +236,47 @@ if ( NOT ${BUILDINSRC} )
 			    PATTERN "*.ico" PATTERN "*.rc" PATTERN "*.txt"
 			    EXCLUDE PATTERN CMakeFiles EXCLUDE )
 endif()
+
+if ( APPLE )
+
+	set(LIBRARY_DESTINATION "${CMAKE_BINARY_DIR}/Contents/Frameworks/debug")
```

Comment:
No, use "${CMAKE_BINARY_DIR}/${OD_LIBRARY_DIRECTORY}" as the destination, to make it configuration specific